edg_acoustics/acoustics_simulation.py

```diff
@@ -1055,7 +1143,60 @@ def init_TimeIntegration(self, TI: edg_acoustics.TimeIntegrator, rec: numpy.ndar
             print(f"P at mic locations {self.prec[:,StepIndex]}")
   
 
-        
+    def L(self, P, Vx, Vy, Vz):
```

Comment:
Same as for above-mentioned "step_dt_new"., apart from P0 and V0, we also need to pass self.BC, since those auxiliary varibles also need to be time integrated and they are part of the righ-hand-side of the governing equations.